elements/list-container/list-container.js

```diff
@@ -4,39 +4,49 @@
         count: 13,
 
         ready: function() {
-            this.data = this.generateData();
-            window.list = this.$.list;
-        },
-
-        generateData: function() {
-            var data = [];
-
-            // TODO: Add dynamic dates or easier date formatting
-            var dates = [
-                    "01/12/2015",
-                    "01/19/2015",
-                    "01/26/2015",
-                    "02/02/2015",
-                    "02/09/2015",
-                    "02/16/2015",
-                    "02/23/2015",
-                    "03/02/2015",
-                    "03/09/2015",
-                    "03/16/2015",
-                    "03/23/2015",
-                    "03/30/2015"
-                ];
-
-            for (var i=1; i < dates.length; i++) {
-                // TODO: replace hardcoded data as needed with dynamic data
-                data.push({
-                    id: i,
-                    name: ("Web Components Weekly #" + i),
-                    url: "/" + i,
-                    date: dates[i]
-                });
+          var self = this;
+          this.data = [];
+          var data = [
+        		{
+        			"name": "Web Components Weekly #1",
+        			"id": "5b537c8ca0"
+        		},
+        		{
+        			"name": "Web Components Weekly #2",
+        			"id": "3d95131149"
+        		},
+        		{
+        			"name": "Web Components Weekly #3",
+        			"id": "6bb0920ff6"
+        		},
+        		{
+        			"name": "Web Components Weekly #4",
+        			"id": "6f50177c1d"
+        		},
+        		{
+        			"name": "Web Components Weekly #5",
+        			"id": "c2d3bdb50a"
+        		}
+        	];            
+
+            for (var i = 0; i < data.length; i++) {
```

Comment:
Hi @Nevraeka, what you think about performance of this array length. My thoughts about this: http://browserdiet.com/en/#cache-array-lengths

It would be better, I can send a pull request chanching the iterator ;)